README.md

```diff
@@ -25,7 +25,7 @@ See also [Removing the Scheduler](docs/Installation.md#uninstallation)
 - [The server time must be UTC](#server-time)
 - Replicas that host jobs must be configured as a [readable secondary][readable-secondary]
 - All requisite DBs must be created and added to the AG before installation.
-- Deployment scripts use integrated security. Use of SQL Logins has not been tested but can be attempted but adding the appropriate [`Invoke-SqlCmd`][invoke-sqlcmd - bol] flags in the [tsqlScheduler module](src/tsqlScheduler/tsqlScheduler.psm1).
+- Deployment scripts use integrated security. Use of SQL Logins has not been tested but can be attempted but adding the appropriate [`Invoke-SqlCmd`][Invoke-SqlCmd -TrustServerCertificate - bol] flags in the [tsqlScheduler module](src/tsqlScheduler/tsqlScheduler.psm1).
```

Comment:
Find/replace overzealousness 😛  No need to update the link name.

```suggestion
- Deployment scripts use integrated security. Use of SQL Logins has not been tested but can be attempted but adding the appropriate [`Invoke-SqlCmd`][Invoke-SqlCmd - bol] flags in the [tsqlScheduler module](src/tsqlScheduler/tsqlScheduler.psm1).
```

Reply:
Thank you Peter. I have fixed it